Programs/ReactorControl/LaserControllerExe.lua

```diff
@@ -0,0 +1,13 @@
+os.loadAPI("\\Api\\MekanismReactor\\LaserController.lua")
+os.loadAPI("\\Api\\Communication\\MessageCommunicator.lua")
+
+local obj_com = MessageCommunicator.MessageCommunicator:new("right")
+local obj_controller = LaserController.LaserController:new(
```

Comment:
consider moving the executables for the interface into the interfaces as a min routine. is less complicated then